### PR TITLE
Version 2: Add gh-pages

### PR DESCRIPTION
**Pull request by OleksandrSPV**
Added Github pages package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +68,8 @@
     "storybook": "storybook dev -p 6006",
     "test:ci": "vitest run --root src/",
     "test:unit": "vitest --root src/",
-    "coverage": "vitest run --coverage --root src/"
+    "coverage": "vitest run --coverage --root src/",
+    "predeploy": "yarn build",
+    "deploy:gh-pages": "gh-pages -d dist"
```
Comment (donskov) [inline package.json:73]: @OleksandrSPV I Just tested the `deploy:gh-pages` command and the `predeploy` command didn't run.

**Review by donskov — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (OleksandrSPV) [reply package.json:73]: > @OleksandrSPV I Just tested the `deploy:gh-pages` command and the `predeploy` command didn't run.

hm, let's try npm, I'll make changes

Comment (donskov) [reply package.json:73]: It's not related to the npm. I think the problem is the `deploy:gh-pages` command name. It must be `deploy`.

Comment (OleksandrSPV) [reply package.json:73]: yes, you are right, thanks

Comment (OleksandrSPV) [reply package.json]: done @donskov 
thank you for your attention

**Pull request merged**